lib/mix/tasks/timber/install/endpoint_file.ex

```diff
@@ -4,7 +4,7 @@ defmodule Mix.Tasks.Timber.Install.EndpointFile do
   alias Mix.Tasks.Timber.Install.FileHelper
 
   def update!(file_path, api) do
-    router_pattern = ~r/( *)plug [^\n\r]*.\.Router/
+    router_pattern = ~r/( *)plug.{0,1}[^\n\r]*.\.Router.{0,1}/
```

Comment:
Would it make sense to specify that it should be an open and close parenthesis by doing something like: `( *)plug\({0,1}[^\n\r]*.\.Router\){0,1}`?

Reply:
You are right. It's more explicit that way. Feel free to open another PR to avoid an extra commit.